### DISCUSSION TITLE
[LICENSE] Dependency `codesandbox-import-util-types` is under LGPL-3.0

### DISCUSSION THREAD

**Discussion by Geminii** [codesandbox/sandpack#1073] [General]
Hi Sandpack team 👋 

We are using the `sandpack-react` dependency which is under **Apache 2.0 license** but regarding the `package.json`: https://github.com/codesandbox/sandpack/blob/3bfd4acfddd3139629b71dedbce40d24cdf96673/sandpack-react/package.json#L68 you are using `codesandbox-import-util-types` which is under **LGPL-3.0 license**.

It seems that this dependency is only used to get the `ITemplate` type. 
Could it possible to inject this type directly inside the `sandpack` library to be fully **Apache 2.0 license** ? 

Available to open a PR if needed or open for you suggestion 🙂
Thanks a lot 🙏

**Comment by Geminii** [comment]
Sorry about duplication of https://github.com/codesandbox/sandpack/discussions/984 🙈

**Comment by danilowoz** [comment]
Hey! We should do it. I can't see any reason why we still need to use those packages.
Would you like to open a PR for that? This has been quite week, so I can't make any promises about when I'd have time to address it

**Reply by Geminii** [reply]
@danilowoz Thanks for you quick answer 🙏 
I will soon open a PR to tackle this subject 👍

**Reply by Geminii** [reply]
@danilowoz #1075 is available 🙂
Thanks for your review 🙏

**Reply by Geminii** [reply]
Just to follow up on this discussion, the PR #1075  has been merged and a release has already been made 🔥 

Thank you very much @danilowoz  for your responsiveness 🙏
